### PR TITLE
80 update collab list

### PR DESCRIPTION
**Pull request by luke-m**

### CODE REVIEW

File: src/components/BioCollaboratorTile/template.ts [hunk 1]
```diff
@@ -10,10 +11,38 @@ export default (
...
+						</svg>
+						GitHub
+					</a>
+				</div>
+                <img src="${data.imgsrc}" />
+            </div>
             <p>
                 <slot></slot>
             </p>
+            ${
```
Comment (IvailoStoyanov) [inline src/components/BioCollaboratorTile/template.ts]: Can you please write this on one line

File: src/components/BioCollaboratorTile/template.ts [hunk 2]
```diff
@@ -10,10 +11,38 @@ export default (
 	return render`
         ${createStyle(styles)}
         <div class="wrapper">
-            <img src="${data.imgsrc}" />
+            <div class="img-wrapper">
+				<div class="info">
+					<a target="_blank" rel="noopener noreferrer" href="${data.github}">
```
Comment (IvailoStoyanov) [inline src/components/BioCollaboratorTile/template.ts]: Perhaps link the icons to the profiles?

File: src/scaffolding/master.scaffolding.hbs [hunk 3]
```diff
@@ -246,7 +246,7 @@
     </div>
 </footer>
 
-<bio-modal modalType="whosBehind" data-resources="[{paths: ['components/BioModal/index.js']}]">
+<bio-modal modalType="whosBehind" openonpageload="true" data-resources="[{paths: ['components/BioModal/index.js']}]">
```
Comment (IvailoStoyanov) [inline src/scaffolding/master.scaffolding.hbs]: why do you want it open on page load ?

File: src/_assets/link.svg [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<svg version="1.1" id="Capa_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 482.8 482.8" style="enable-background:new 0 0 482.8 482.8; fill: white" xml:space="preserve">
+<g>
+	<g>
+		<path d="M255.2,209.3c-5.3,5.3-5.3,13.8,0,19.1c21.9,21.9,21.9,57.5,0,79.4l-115,115c-21.9,21.9-57.5,21.9-79.4,0l-17.3-17.3
```
[the remark this answers is not in the document]
Comment (luke-m) [reply src/_assets/link.svg]: I am manipulating the `fill`

**Pull request merged**